### DISCUSSION TITLE
How to test with browser resolution width below 380px

### DISCUSSION THREAD

**Discussion by lboradzhiev** [webdriverio/webdriverio#13248] [Community]
Hey guys I need to make a test for which browser width should be 380px and just could not manage to do so 
I've tried :
 - to put resolution in capabilities 
 - browser. setWindowSize
  but nothing works
Also I

**Comment by christian-bromann** [comment]
Hey @lboradzhiev 👋 browser don't allow you to scale the viewport smaller than 500px. If you'ld like to emulate a mobile screen resolution, I recommend to checkout the [`emulateDevice`](https://webdriver.io/docs/devtools-service#device-emulation) of the devtools service. Let me know if you have any further questions.
